image/templates/helm/stackrox-secured-cluster/internal/defaults/50-images.yaml.htpl

```diff
@@ -33,13 +31,12 @@ image:
     registry: {{ ._rox.image.registry }}
   collector:
     registry: {{ if or (eq ._rox.image.registry "stackrox.io") (eq ._rox.image.registry "registry.connect.redhat.com") }}collector.stackrox.io{{ else }}{{ ._rox.image.registry }}{{ end }}
+  # TODO: Shouldn't the [<- if not .KubectlOutput >] check be here?
```

Comment:
>  TODO: Shouldn't the [<- if not .KubectlOutput >] check be here?

IMO probably should be, however the image references defined in this file still need to be used - which doesn't seem like they are - so not 'technically' breaking anything.

Reply:
Moritz says it should stay as-is due to node scanning